### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by DavidBM** [bnoordhuis/v8.rs#1]
If yes, what is its state? 
If not, do you know other similar project?

**Comment by bnoordhuis** [comment]
It's pretty dead.  I wrote it to scratch an itch but I ended up solving it differently.

Maybe try https://github.com/rustbridge/neon?

**Comment by DavidBM** [comment]
Thanks! 

But I'm searching a way to execute a .js file from rust code. The idea is to have a server in Rust and let user to write plugins in JS.

**Comment by bnoordhuis** [comment]
Perhaps you can integrate https://github.com/servo/rust-mozjs somehow?

**Comment by DavidBM** [comment]
Ohhh.
Yes! I will research it. Thank you very much! :D

**Comment by xoftware** [comment]
David, any luck? I'm considering Rust with JS. Memory management, framework, performance, and concurrency looks great. Interesting combination, since JavaScript is Garbage Collected and Single-threaded, yet Rust is Owned/Borrowed and Multi-threaded. Both high-level Syntax, though.

**Comment by DavidBM** [comment]
Sorry  @xoftware but I stopped my project. But the best shoot (I think) is the spider monkey engine. They are using it now for servo browser, then I think you can use it. @bnoordhuis gave me the link here. Maybe you can see how servo implement it. 

Resources:
https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey
https://github.com/servo/rust-mozjs
https://github.com/servo/servo

**Comment by sergeysova** [comment]
@xoftware what about emscripten with rust?

**Comment by DavidBM** [comment]
I don't see what is the point on that @LestaD. The project want to run js inside rust, no rust inside js. That means, Rust will get js code and will run it. Emscripten get the rust code and compile to a subset of js.

**Comment by jcbhmr** [comment]
I recently stumbled across this v8.rs repository while looking for V8's C++ bindings exposed to Rust-land so that I could do V8 C++ stuff from Rust across the Rust/C++ boundary. I eventually found the Deno rusty_v8 project https://github.com/denoland/rusty_v8 which did what I wanted. I think adding a link to https://github.com/denoland/rusty_v8 on the readme and/or archiving this repository to indicate its current status would be a good idea to point other Google travelers to the right place.
